bazel/repositories.bzl

```diff
@@ -19,6 +19,17 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 def proxy_wasm_cpp_host_repositories():
     # Bazel extensions.
 
+    # Update platforms for crate_universe. Can remove when we update Bazel version.
+    maybe(
+        http_archive,
+        name = "platforms",
+        urls = [
+            "https://mirror.bazel.build/github.com/bazelbuild/platforms/releases/download/0.0.10/platforms-0.0.10.tar.gz",
+            "https://github.com/bazelbuild/platforms/releases/download/0.0.10/platforms-0.0.10.tar.gz",
+        ],
+        sha256 = "218efe8ee736d26a3572663b374a253c012b716d8af0c07e842e82f238a0a7ee",
+    )
+
```

Comment:
Similarly to the abseil comment in #400, this is not a dependency of C++ Host, so it should be fixed (or at least attempted to) by updating `rules_rust` to a more recent version, which updates `platforms`.

Having said that, if it's more than a simple version bump, then perhaps it's better to do that in a follow-up PR, to avoid adding extra complexity here.

Reply:
> if it's more than a simple version bump

Yes it is. A Bazel update at least, and I haven't gotten it working yet.

Reply:
Ack.